src/deck-options.md

```diff
@@ -88,29 +94,17 @@ Controls how many new cards are introduced each day you open the program. If you
 study fewer than the limit, or miss a day, the next day the counts will be back
 to your limit - they do not accumulate.
 
-In [the v3 scheduler](https://faqs.ankiweb.net/the-2021-scheduler.html#daily-limits), Anki
-uses the limits of the deck you select as an upper limit on the number of cards.
-This means if "French" has a limit of 20 cards and "French::Lesson 1" and
-"French::Lesson 2" both have limits of 15 cards, when you click on "French",
-you’ll get at most 15 cards from either child deck, and only 20 cards in total.
-
-In the v1 or v2 scheduler, each parent applies its limits to its children. If you
-have decks in a grandparent-parent-child arrangement, both the grandparent and
-parent limits will alter how many cards are shown from the child, even if you
-click directly on the child. In the v3 scheduler, this has been simplified, and
-now each deck's limit controls how many cards from that specific deck can be used.
-Imagine that you have this collection:
-
-    - Parent deck (New Cards/Day: 20)
-      - Child deck (New Cards/Day: 7)
-        - Grandchild deck A (New Cards/Day: 5)
-          - Cards 1 to 50
-        - Grandchild deck B (New Cards/Day: 5)
-          - Cards 51 to 100
-
-- If you click on the Parent deck, you will get 10 new cards (from Grandchild decks A and B).
-- If you click on the Child deck, you will get 7 new cards (from Grandchild decks A and B).
-- If you click on any of the Grandchild decks, you will get 5 new cards.
+Each parent applies its limits to its children, but schedulers 1, 2 and 3 behave differently: 
+- In the v1 and v2 schedulers, if you
+have decks in a parent-child-grandchild arrangement, both the parent and
+child limits will alter how many cards are shown from the grandchild, **even if you
+click directly on the grandchild.**
+
+- In the v3 scheduler, intermediate deck limits
+also affect their children, but if you click directly on the grandchild deck, 
+child and parent limits will be ignored. Please see the 
+[the v3 scheduler](https://faqs.ankiweb.net/the-2021-scheduler.html#daily-limits)
+page for more info.
```

Comment:
These two paragraphs feel a bit confusing, and I'm not sure the comment about v2 is entirely correct. How does the following sound?

When decks are nested (eg `parent`, `parent::child`, `parent::child::grandchild`), the way the limits are applied depends on the scheduler version.

- V1 applies parent limits to children, regardless of which deck you click on
- V2 behaves similarly to V1 for new cards. For reviews, only the limits of the deck you click on are honored.
- V3 honors the limits of the deck you click on, and any decks inside it. Limits from parents above the deck you clicked on are ignored.

For more information, please see the 
[the v3 scheduler](https://faqs.ankiweb.net/the-2021-scheduler.html#daily-limits) page.

Reply:
Done (I wasn't sure about the v2, thanks for clarifying!)